scripts/hostcfgd

```diff
@@ -156,11 +158,15 @@ class Feature(object):
             feature_cfg (dict): Feature CONFIG_DB configuration
             deviec_config (dict): DEVICE_METADATA section of CONFIG_DB
         """
+        if 'has_timer' in feature_cfg:
```

Comment:
hi @dgsudharsan , wonder why you don't want 'has_timer' in config_db.json, FEATURE section?

Reply:
@wenyiz2021 Please refer to this comment https://github.com/sonic-net/sonic-host-services/pull/45#discussion_r1119395605

Reply:
Yes. If the config is part of upgrade, it will be automatically modified by the db_migrator. However if the user explicitly sets has_timer and performs a config reload in an image, it is expected to fail.

Reply:
thanks for clarifying